tests/test_sofastream.py

```diff
@@ -57,3 +57,70 @@ def test_sofastream_inspect(capfd, temp_sofa_file):
     with open(inspect_file, "r") as out_inspect:
         text = out_inspect.readlines()
     assert out == "".join(text)
+
+
+def test_list_dimensions(capfd, tmp_path_factory):
```

Comment:
we could also add a test which compares the output with the output from the ``Sofa`` class. because they should behave in the same way

Reply:
In this case they don't. The SOFA class also tells you which variable sets the dimension. However, this is not required for read only access and would require parsing the conventions files. Hence, we decided to not implement it here.